src/core/ext/filters/client_channel/subchannel_index.cc

```diff
@@ -26,14 +26,24 @@
 #include <grpc/support/alloc.h>
 #include <grpc/support/string_util.h>
 
+#include "src/core/ext/filters/client_channel/subchannel.h"
 #include "src/core/lib/avl/avl.h"
 #include "src/core/lib/channel/channel_args.h"
 #include "src/core/lib/gpr/tls.h"
+#include "src/core/lib/iomgr/exec_ctx.h"
+#include "src/core/lib/iomgr/timer.h"
+
+// Sweeps unused subchannels every 10 seconds.
```

Comment:
I think we need more documentation on this GC semantics: why are we doing sweeps, what are the guarantees regarding what will be collected, why we chose the given interval, etc.